app/pages/sections/routes.js

```diff
@@ -184,6 +188,10 @@ export const sectionRoutes = (authProvider, addContext, sessionManager) => {
     const { orderId } = req.params;
     const accessToken = extractAccessToken({ req, tokenType: 'access' });
     await getFundingSource({ orderId, accessToken });
+    let description = sessionManager.getFromSession({ req, key: 'description' });
+    if (!description) {
+      description = await getOrderDescription({ orderId, accessToken });
+    }
```

Comment:
This can be replaced by a single call to the new getOrderDescription function. You'll have to pass some additional parameters though, `req` (instead of `orderId`), `sessionManager`, and `logger` if I remember correctly.